extensions/amp-orientation-observer/0.1/test/validator-amp-orientation-observer.out

```diff
@@ -0,0 +1,121 @@
+|    See the License for the specific language governing permissions and
+|    limitations under the license.
+|  -->
+|  <!--
+|    Test Description:
+|    Tests amp-orientation-observer.
+|  -->
+|  <!doctype html>
+|  <html ⚡>
+|  <head>
+|    <meta charset="utf-8">
+|    <link rel="canonical" href="./regular-html-version.html">
+|    <meta name="viewport" content="width=device-width,minimum-scale=1">
+|    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+|    <script async src="https://cdn.ampproject.org/v0.js"></script>
+|    <script async custom-element="amp-orientation-observer" src="https://cdn.ampproject.org/v0/amp-orientation-observer-0.1.js"></script>
+|  </head>
+|  <body>
+|    <!-- Valid -->
+|    <amp-orientation-observer layout="nodisplay"></amp-orientation-observer>
+|    <amp-orientation-observer layout="nodisplay" on="alpha:foo;beta:bar;gamma:baz(percent=event.percent)"></amp-orientation-observer>
+|    <amp-orientation-observer layout="nodisplay" on="alpha:foo;beta:bar;gamma:baz(angle=event.angle)"></amp-orientation-observer>
+|    <amp-orientation-observer layout="nodisplay" on="alpha:foo(percent=event.percent);beta:bar;gamma:baz"></amp-orientation-observer>
+|    <amp-orientation-observer layout="nodisplay" on="alpha:foo(angle=event.angle);beta:bar;gamma:baz"></amp-orientation-observer>
+|    <amp-orientation-observer layout="nodisplay" on="alpha:foo;beta:bar(percent=event.percent);gamma:baz"></amp-orientation-observer>
+|    <amp-orientation-observer layout="nodisplay" on="alpha:foo;beta:bar(angle=event.angle);gamma:baz"></amp-orientation-observer>
+|    <amp-orientation-observer layout="nodisplay" gamma-range="0 90" on="alpha:foo;beta:bar;gamma:baz(percent=event.percent)"></amp-orientation-observer>
+|    <amp-orientation-observer layout="nodisplay" gamma-range="0 90" on="alpha:foo;beta:bar;gamma:baz(angle=event.angle)"></amp-orientation-observer>
+|    <amp-orientation-observer layout="nodisplay" alpha-range="0 90" on="alpha:foo(percent=event.percent);beta:bar;gamma:baz"></amp-orientation-observer>
+|    <amp-orientation-observer layout="nodisplay" alpa-range="0 90" on="alpha:foo(angle=event.angle);beta:bar;gamma:baz"></amp-orientation-observer>
```

Comment:
nit: you have an invalid in your valid list

Reply:
Fixed.